dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/programrule/implementers/event/ShowErrorWarningExecutorTest.java

```diff
@@ -82,14 +83,14 @@ class ShowErrorWarningExecutorTest extends DhisConvenienceTest
     private final static String ANOTHER_DATA_ELEMENT_ID = "AnotherDataElementId";
 
     private ShowWarningOnCompleteExecutor warningOnCompleteExecutor = new ShowWarningOnCompleteExecutor(
-        getErrorActionRule( WARNING ) );
+        getErrorRuleAction( WARNING ) );
 
     private ShowErrorOnCompleteExecutor errorOnCompleteExecutor = new ShowErrorOnCompleteExecutor(
-        getErrorActionRule( ERROR ) );
+        getErrorRuleAction( ERROR ) );
 
-    private ShowErrorExecutor showErrorExecutor = new ShowErrorExecutor( getErrorActionRule( ERROR ) );
+    private ShowErrorExecutor showErrorExecutor = new ShowErrorExecutor( getErrorRuleAction( ERROR ) );
 
-    private ShowWarningExecutor showWarningExecutor = new ShowWarningExecutor( getErrorActionRule( WARNING ) );
+    private ShowWarningExecutor showWarningExecutor = new ShowWarningExecutor( getErrorRuleAction( WARNING ) );
```

Comment:
Reading `getErrorRuleAction` and passing in a `WARNING` feels odd 🤔 what does this mean?